### PR TITLE
delete book from store if there is no book return an error

### LINKED ISSUE #7: User should be able to delete a book from the store
Add a new function in main.py called `delete_book`
`delete_book` should ask for the book's ID and call the Bookstore `delete_book` function.  Use try-except to display an error message saying "Error: Book Not Found" if the user tries to delete a book that doesn't exist. 

Add a new menu option with the action `delete_book`.  This should be option 7.

Don't ask the user for any confirmation - just delete the book.

### PR DESCRIPTION
**Pull request by xm6264jz**
@yp0975eu  Now this code works. Have a look at it and review. Thanks

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def create_menu():
     menu.add_option('4', 'Show Read Books', show_read_books)
     menu.add_option('5', 'Show All Books', show_all_books)
     menu.add_option('6', 'Change Book Read Status', change_read)
+  
+
```
Comment (yp0975eu) [inline main.py]: @xm6264jz , There's no delete option here.

**Review by yp0975eu — CHANGES_REQUESTED** [review]
This pull request doesn't look complete.

**Comment by xm6264jz** [comment]
@yp0975eu  I will make a change to that

**Comment by xm6264jz** [comment]
@yp0975eu you select it through the menu option. There is a second commit. that makes it 2 commits

**Comment by xm6264jz** [comment]
@yp0975eu I get it now where the mistake is... I will make a change to it

File: main.py [hunk 2]
```diff
@@ -26,6 +26,13 @@ def create_menu():
     menu.add_option('4', 'Show Read Books', show_read_books)
     menu.add_option('5', 'Show All Books', show_all_books)
     menu.add_option('6', 'Change Book Read Status', change_read)
+    menu.add_option('Q', 'Quit', quit_program)
```
Comment (yp0975eu) [inline main.py]: There are two Q options. 1 on line 29 and one on line 36

**Review by yp0975eu — COMMENTED** [review]
There's a duplicate option `Q`. Can you remove the one you added on line 29 and move the one on line 36 directly underneath option 7?

**Comment by xm6264jz** [comment]
@yp0975eu I have deleted one of the Q options.

File: bookstore.py [hunk 3]
```diff
@@ -194,6 +194,9 @@ def get_book_by_id(self, id):
             
             if book_data:
                 book = Book(book_data['title'], book_data['author'], book_data['read'], book_data['rowid'])
+
+
+
```
Comment (yp0975eu) [inline bookstore.py:200]: Line 197 - 199 white space is unnecessary

**Review by yp0975eu — APPROVED** [review]

**Comment by yp0975eu** [comment]
@xm6264jz, I've approved this pull request.

**Pull request merged**